openshift/template.yaml

```diff
@@ -1,2 +1,171 @@
+            value: "900"
+          - name: HPF_SCORING_REGION
+            value: "${HPF_SCORING_REGION}"
+          - name: USE_FILTERS
+            value: "${{USE_FILTERS}}"
+          - name: AWS_S3_ENDPOINT_URL
+            value: ""
+          - name: AWS_S3_BUCKET_NAME
+            value: "hpf-insights"
+          - name: FLASK_LOGGING_LEVEL
+            value: ${FLASK_LOGGING_LEVEL}
+          - name: AWS_S3_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                key: s3-access-key-id
+                name: aws
+          - name: AWS_S3_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                key: s3-secret-access-key
+                name: aws
+          name: bayesian-hpf-insights
+          image: "${DOCKER_REGISTRY}/${DOCKER_IMAGE}:${IMAGE_TAG}"
+          ports:
+          - containerPort: 6006
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 6006
+            initialDelaySeconds: 30
+            periodSeconds: 60
+            timeoutSeconds: 30
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 6006
+            initialDelaySeconds: 30
+            periodSeconds: 60
+            timeoutSeconds: 30
+          resources:
+            requests:
+              cpu: ${CPU_REQUEST}
+              memory: ${MEMORY_REQUEST}
+            limits:
+              cpu: ${CPU_LIMIT}
+              memory: ${MEMORY_LIMIT}
+          #Add so that .npy can be downloaded from s3 and saved under /tmp. Gives write permission error otherwise.
+          volumeMounts:
+          - name: s3-data
+            mountPath: /tmp
+        volumes:
+        - name: s3-data
+          emptyDir: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      service: bayesian-hpf-insights-${HPF_SCORING_REGION}
+    name: bayesian-hpf-insights-${HPF_SCORING_REGION}
+  spec:
+    ports:
+    - port: 6006
+      targetPort: 0
+    selector:
+      service: bayesian-hpf-insights-${HPF_SCORING_REGION}
+
+parameters:
+- description: CPU request
+  displayName: CPU request
+  required: true
+  name: CPU_REQUEST
+  value: "500m"
+
+- description: CPU limit
+  displayName: CPU limit
+  required: true
+  name: CPU_LIMIT
+  value: "1000m"
+
+- description: Memory request
+  displayName: Memory request
+  required: true
+  name: MEMORY_REQUEST
+  value: "1024Mi"
+
+- description: Memory limit
+  displayName: Memory limit
+  required: true
+  name: MEMORY_LIMIT
+  value: "2048Mi"
+
+# For testing, image on Docker
+# TODO: Change to registry.devshift
+- description: Docker registry where the image is
+  displayName: Docker registry
+  required: true
+  name: DOCKER_REGISTRY
+  value: "docker.io"
+
+# For testing, image on Docker
+# TODO: image name to follow f8a-image conventions.
+- description: Docker image to use
+  displayName: Docker image
+  required: true
+  name: DOCKER_IMAGE
+  value: "sara02/hpf-insights"
```

Comment:
When you do change the image name, change it to `f8a-{ecosystem}-insights`, which since you're rolling this out for maven would be f8a-maven-insights.

Reply:
Thanks. I will remember the `ecosystem` part as well.